address-controller/src/main/java/io/enmasse/controller/common/KubernetesHelper.java

```diff
@@ -295,6 +296,7 @@ private JsonObject doRawHttpRequest(String path, String method, JsonObject body,
         OkHttpClient httpClient = client.adapt(OkHttpClient.class);
 
         HttpUrl url = HttpUrl.get(client.getOpenshiftUrl()).resolve(path);
+        //log.info("Performing {} on {} with body '{}'", method, path, body.encodePrettily());
```

Comment:
nitpick: probably shouldn't add commented out code, if the log is potentially useful we could keep it at debug level